### DISCUSSION TITLE
Stuck on "Loading..." screen loop at Purchase Orders section

### DISCUSSION THREAD

**Discussion by wafflecrate** [inventree/InvenTree#6202] [General]
Version info:
# Version Information:
InvenTree-Version: 0.12.10
Django Version: 3.2.19
Commit Hash: cc29f39
Commit Date: 2023-11-22

The problem here is that when I navigate into Buy > Purchase Orders section it no longer loads and shows the existing purchase orders. Instead it is stuck on a neverending "Loading" screen. I've tried logging out, then back in. Also tried logging in using a different Wifi Internet connection.

I've barely used the Purchase Order functionality and the "13" rows it shows are mostly the demo purchase orders. I discovered this issue when I tried to create a real purchase order for the first time.

Any ideas as to what might be going on?

![Inventree error](https://github.com/inventree/InvenTree/assets/67816847/54882875-a791-45a5-9c67-e4ac88576783)

**Comment by SchrodingersGat** [comment]
That likely means that there has been an error loading the table data. Can you share the developer console logs in your browser? It should show either an error with the frontend (javascript) or an API error

**Reply by wafflecrate** [reply]
Is that the same thing Errors under the "ERROR_REPORT" section in the Inventree Admin screen or is a different screen?

**Reply by SchrodingersGat** [reply]
No, in the web browser itself. It will have a "developer console" which will show any errors:

https://balsamiq.com/support/faqs/browserconsole/

**Reply by wafflecrate** [reply]
Got it. Is there a private way that I can share the logs?

**Reply by SchrodingersGat** [reply]
The javascript console logs should be reasonable anonymous. However if there is anything sensitive in there, you can email `support at inventree dot org`

**Reply by wafflecrate** [reply]
Sent!

**Reply by SchrodingersGat** [reply]
@wafflecrate thanks. From the logs, this is a bug present in `0.12.10` which has subsequently been fixed in `0.13.0` - please update!
